website/content/v1.5/introduction/prodnotes.md

```diff
@@ -99,10 +99,11 @@ https://kube.cluster1.mydomain.com:6443
 
 ## Load balancing the Talos API
 
-The `talosctl` tool provides built-in client-side load-balancing across control plane nodes, if they are all reachable from the workstation where you run `talosctl`.
-So usually you do not need to configure a load balancer for the Talos API.
+The `talosctl` tool provides built-in client-side load-balancing across control plane nodes, so usually you do not need to configure a load balancer for the Talos API.
 
-If the control plane nodes are not directly reachable from the workstation where you run `talosctl`, then configure a load balancer to forward TCP port 50000 to the control plane nodes.
+However, if the control plane nodes are *not* directly reachable from the workstation where you run `talosctl`, then configure a load balancer to forward TCP port 50000 to the control plane nodes.
+
+> Note: Because the Talos Linux API uses gRPC and mutual TLS, it cannot be proxied by a HTTP/S proxy, but only by a TCP load balancer.
```

Comment:
We should probably add same note for Kubernetes API, even though it's not mutual TLS, but TLS termination is done by the `kube-apiserver` directly, so it should be TCP loadbalancer always.